Sources/SwiftParser/StringLiteralRepresentedLiteralValue.swift

```diff
@@ -71,8 +71,14 @@ extension StringSegmentSyntax {
   ) {
     precondition(!hasError, "appendUnescapedLiteralValue relies on properly parsed literals")
 
-    var text = content.text
-    text.withUTF8 { buffer in
+    let rawText = content.rawText
+    if !rawText.contains("\\") {
+      // Fast path. No escape sequence.
+      output.append(String(syntaxText: rawText))
+      return
+    }
```

Comment:
This first path is an addition. It's _probably_ just faster than always lexing the UTF8 sequence.